Manifests/ManagedPreferencesApplications/com.github.macadmins.SupportCompanion.plist

```diff
@@ -612,26 +655,27 @@
 		</dict>
 		<dict>
 			<key>pfm_description</key>
-			<string>Configures the level of information to show on the desktop. Defaults to Full.</string>
+			<string>Configures the level of information to show on the desktop. Defaults to 4.</string>
 			<key>pfm_name</key>
 			<string>DesktopInfoLevel</string>
 			<key>pfm_range_list</key>
 			<array>
-				<string>Full</string>
-				<string>Minimal</string>
-				<string>Hardware</string>
-				<string>Custom</string>
+				<integer>1</integer>
+				<integer>2</integer>
+				<integer>3</integer>
+				<integer>4</integer>
+				<integer>5</integer>
 			</array>
 			<key>pfm_title</key>
 			<string>Desktop Info Level</string>
 			<key>pfm_type</key>
-			<string>string</string>
+			<string>integer</string>
 		</dict>
 		<dict>
 			<key>pfm_description</key>
-			<string>If DesktopInfoLevel is set to Custom, use this array to determine which information to show. Available info are: HostName, Model, SerialNumber, Processor, IpAddress, MemSize, OsBuild, OsVersion, LastBootTime, StorageInfo, SupportPhone, SupportEmail, Separator.</string>
+			<string>Use this array to determine which information to hide. Available items are: HostName, Model, SerialNumber, Processor, IPAddress, Memory, OSBuild, OSVersion, LastRestart, FileVault, StorageName, SupportPhone, SupportEmail. It is also possible to hide entire sections: Hardware Specifications, System Information, Network Information, Storage, Support.</string>
 			<key>pfm_name</key>
-			<string>DesktopInfoCustomItems</string>
+			<string>DesktopInfoHideItems</string>
```

Comment:
The string subkey of this array could have a `pfm_range_list` with the available items. You can then also add a `pfm_range_list_titles` for display-friendly strings (e.g. 'IP Address' instead of 'IpAddress').